### ISSUE TITLE
Server Crash

### ISSUE THREAD

**Issue by rbdannenberg** [rbdannenberg/soundcool#69]
Select Project:New
Click on Player icon (top left)
Click on Speaker icon (row 6, right)
Connect Player to Speaker
Click Player's Snd button
Select sound.wav
(See that Player shows sound.wav as loaded)
Click play button.
(Note: no sound)
Project:Save As
Enter project name test, description test
(Note: Server crashes now:
```
{ location:
   'https://freesound.org/data/previews/210/210533_985466-lq.mp3' }
/Users/rbd/soundcool/soundcool/src/backend/node_modules/mysql/lib/protocol/Parser.js:437
      throw err; // Rethrow non-MySQL errors
      ^

Error: ENOENT: no such file or directory, stat './uploads/sounds/1582037622837-sound.wav'
    at Object.statSync (fs.js:855:3)
    at Query.connection.query (/Users/rbd/soundcool/soundcool/src/backend/routes/sounds.js:368:23)
    at Query.<anonymous> (/Users/rbd/soundcool/soundcool/src/backend/node_modules/mysql/lib/Connection.js:525:10)
    at Query._callback (/Users/rbd/soundcool/soundcool/src/backend/node_modules/mysql/lib/Connection.js:491:16)
    at Query.Sequence.end (/Users/rbd/soundcool/soundcool/src/backend/node_modules/mysql/lib/protocol/sequences/Sequence.js:83:24)
    at Query._handleFinalResultPacket (/Users/rbd/soundcool/soundcool/src/backend/node_modules/mysql/lib/protocol/sequences/Query.js:139:8)
    at Query.EofPacket (/Users/rbd/soundcool/soundcool/src/backend/node_modules/mysql/lib/protocol/sequences/Query.js:123:8)
    at Protocol._parsePacket (/Users/rbd/soundcool/soundcool/src/backend/node_modules/mysql/lib/protocol/Protocol.js:291:23)
    at Parser._parsePacket (/Users/rbd/soundcool/soundcool/src/backend/node_modules/mysql/lib/protocol/Parser.js:433:10)
    at Parser.write (/Users/rbd/soundcool/soundcool/src/backend/node_modules/mysql/lib/protocol/Parser.js:43:10)
[nodemon] app crashed - waiting for file changes before starting...
```

**Comment by im-Amitto** [comment]
The server does not have the sound file but an entry exists in the database. I will add a catch so a message will appear instead of a crash. Also, we can add a button that can sync the database with sound files.

**Comment by im-Amitto** [comment]
Fixed in 8976cb9
